test/Streamly/Test/Internal/Data/Fold.hs

```diff
@@ -2,26 +2,175 @@ module Main (main) where
 
 import qualified Streamly.Prelude as S
 import Streamly.Internal.Data.Fold
+import qualified Streamly.Data.Fold as FL
 
 import Test.Hspec.QuickCheck
 import Test.QuickCheck (Property, forAll, Gen, vectorOf, arbitrary, choose)
 import Test.QuickCheck.Monadic (monadicIO, assert, run)
-
+import Test.QuickCheck.Arbitrary
 import Test.Hspec as H
 
 maxStreamLen :: Int
 maxStreamLen = 1000
 
+intMin = minBound :: Int
+intMax = maxBound :: Int
```

Comment:
You might as well inline these. Up to you though.

Reply:
These are tiny functions and this is test code, leave the inlining decision to GHC.

Reply:
By inline, I was talking about literally inlining these.